Source/ALSV4_CPP/Public/Character/ALSPlayerCameraManager.h

```diff
@@ -73,4 +75,7 @@ class ALSV4_CPP_API AALSPlayerCameraManager : public APlayerCameraManager
 
 	UPROPERTY(EditDefaultsOnly, BlueprintReadWrite, Category = "ALS|Camera")
 	FVector DebugViewOffset;
+
+private:
+	UALSDebugComponent* m_debugComponent = nullptr;
```

Comment:
Please use camel case for variable names. `DebugComponent` should be fine.

Reply:
Changed